.github/workflows/pr.yml

```
@@ -55,3 +55,9 @@ jobs:
           else
             ./AudioDelay/bin/Debug/net8.0/AudioDelay
           fi
+          
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
```

actionlint (1.7.7)

missing input "path" which is required by action "actions/upload-artifact@v4". all required inputs are "path"

(action)